guides/V1_space_template_main.md

```diff
@@ -0,0 +1,21 @@
+---
+copyright: 'Copyright IBM Corp. 2018'
+link: 'space-template-main'
+is: 'experimental'
+---
+
+# Space Template
+
+In Watson Work Services a space template is a template for the creation of a space.
+The space template defines user configurable properties for the space, 
+as well as a set of required apps that are automatically added to the space on creation.
+
+# Key fields
+
+| property      | type          | description  |
+| ------------- |:------------- |:-----|
+| properties |SpacePropertyCollection |There are three types of properties in a space template: List, Boolean, and Text.|
+| status |SpaceStatus |Status is a special list style attribute on the template. Status consists of a list of values that can be used to describe the state of the space; for example, open, closed, archived.|
```

Comment:
> special list 

What makes it "special" ?  How is this for example differently used by the system than a separate property named "status" ?

Reply:
`SpaceStatus` same as above... need to describe the elements of the structure... we refer to a `list of values` but don't specify if these are String type or some specific object type

Reply:
> How is this for example differently used by the system than a separate property named "status" ?

Right now the main differences are:

- It's a separate top-level field.
- The history of status changes are tracked and can be fetched.

In future we may have additional webhook from certain statuses, e.g. a status to close a space.

Reply:
I think it might be clearer without the word 'special' More information on the status history can be in a separate page. I can reopen: https://github.com/watsonwork/watsonwork-developer-docs/pull/27